client/my-sites/plans/jetpack-plans/product-store/bundles-list.tsx

```diff
@@ -1,9 +1,10 @@
 import { useTranslate } from 'i18n-calypso';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import productButtonLabel from '../product-card/product-button-label';
 import { FeaturedItemCard } from './featured-item-card';
 import { HeroImage } from './hero-image';
 import { useBundlesToDisplay } from './hooks/use-bundles-to-display';
-import { useCreateCheckout } from './hooks/use-create-checkout';
+import { useStoreItemInto } from './hooks/use-store-item-info';
```

Comment:
I think this is a typo? This should be `useStoreItemInfo` (in**f**o), instead of `useStoreItemInto` (in**t**o), right? 😉

Reply:
Doh! 🤦‍♂️ 

Thank you for catching that.